text_formatter_test.go

```diff
@@ -83,6 +84,16 @@ func TestEscaping_DefaultQuoteCharacter(t *testing.T) {
 	}
 }
 
+func TestEscaping_Time(t *testing.T) {
```

Comment:
Should this be specific to `Time`? How about naming it `TestEscaping_Interface` and trying a couple more types?